ppr-ui/src/views/Dashboard.vue

```diff
@@ -190,6 +192,15 @@ export default class Dashboard extends Vue {
     if (this.isRoleStaff || this.isRoleStaffBcol || this.isRoleStaffReg) {
       return this.hasMhrRole && getFeatureFlag('mhr-ui-enabled')
     } else {
+      if (this.getRegTableNewItem.addedReg) {
+        this.snackBarEvent('Registration was successfully added to your table.')
+        setTimeout(() => {
+          const emptyItem: RegTableNewItemI = {
+            addedReg: '', addedRegParent: '', addedRegSummary: null, prevDraft: ''
+          }
+          this.setRegTableNewItem(emptyItem)
+        }, 4000)
```

Comment:
Why such a long delay?

Reply:
Good question.  I am not sure... It is the same delay as Ppr

Reply:
hmm ok then